### DISCUSSION TITLE
When we'll have a new release?

### DISCUSSION THREAD

**Discussion by lu-zero** [BurntSushi/ripgrep#2625] [Q&A] [answered]
The release 13 uses an old jemallocator that does not build on risc-v, would be possible to have a point release with that bumped?

**Comment by BurntSushi** [comment, marked as answer]
See: https://github.com/BurntSushi/ripgrep/blob/master/FAQ.md#release

> does not build on risc-v

Please file a bug with steps for me to reproduce the problem.
